src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -799,4 +799,129 @@
       it { expect(user.home_project.repositories.find_by(name: 'valid_name').repository_architectures.count).to eq(1) }
     end
   end
+
+  describe 'POST #save_distributions' do
```

Comment:
@hennevogel Not closed!... should be merged. It have more things that the colliding things. And we also agreed in having this right now merged and then @ChrisBr can re-use it or get rid of.